integration-test-common/src/test/java/org/apache/gravitino/integration/test/container/ContainerSuite.java

```diff
@@ -231,6 +232,34 @@ public void startKerberosHiveContainer() {
     }
   }
 
+  public void startJdbcHiveContainer(Map<String, String> envVars) {
+    // If you want to enable SQL based authorization, you need both set the
+    // `ENABLE_JDBC_AUTHORIZATION` environment.
+    if (envVars == null
+        || (!Objects.equals(envVars.get(HiveContainer.HIVE_RUNTIME_VERSION), HiveContainer.HIVE3))
+        || (!envVars.containsKey(HiveContainer.ENABLE_AUTHORIZATION))) {
```

Comment:
I think it better to change `HiveContainer.ENABLE_AUTHORIZATION` to `HiveContainer.ENABLE_SQL_BASE_AUTHORIZATION`?

Reply:
Renamed.